lib/urlwatch/command.py

```diff
@@ -380,7 +380,10 @@ def set_error(job_state, message):
                                    'Same Old, Same Old\n'))
         report.error(set_error(build_job('Error Reporting', 'http://example.com/error', '', ''), 'Oh Noes!'))
 
-        report.finish_one(name)
+        reported = report.finish_one(name)
+
+        if not reported:
+            raise ValueError(f'Reporter not enabled: {name}')
```

Comment:
```suggestion
        if not report.finish_one(name):
            raise ValueError(f'Reporter not enabled: {name}')
```

..but is that changing behaviour from before?